flang/lib/Optimizer/CodeGen/TargetRewrite.cpp

```diff
@@ -171,7 +171,6 @@ class TargetRewrite : public TargetRewriteBase<TargetRewrite> {
     } else {
       assert(m.size() == 2);
       // COMPLEX is split into 2 separate arguments
-      auto iTy = rewriter->getIntegerType(32);
```

Comment:
This is used further down, isn't it? 🤞🏻 our new shiny CI is correct here: https://github.com/flang-compiler/f18-llvm-project/actions/runs/1793975819